presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/file/PermissionMetadata.java

```diff
@@ -54,8 +56,14 @@ public boolean isGrantOption()
         return grantOption;
     }
 
+    @JsonProperty
+    public HivePrincipal getGrantee()
+    {
+        return grantee;
+    }
+
     public HivePrivilegeInfo toHivePrivilegeInfo()
     {
-        return new HivePrivilegeInfo(permission, grantOption, new HivePrincipal(USER, "admin"), new HivePrincipal(USER, "admin"));
+        return new HivePrivilegeInfo(permission, grantOption, new HivePrincipal(USER, "admin"), grantee);
```

Comment:
im not convinced this belongs to `Make principal in listTablePrivileges Optional` commit